src/modules/courses/dto/create-document-course.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsNotEmpty, IsOptional, IsString } from 'class-validator';
+export class CreateDocumentByCourseDto {
```

Comment:
chỗ này em sửa lại tên nhé, cả tên file nữa - `[operation]-[module-name]-[specific-target]`
```suggestion
export class CreateCourseDocumentDto {
```